tensorflow_addons/metrics/tests/cohens_kappa_test.py

```diff
@@ -237,3 +237,13 @@ def test_cohen_kappa_serialization():
 
     ck = CohenKappa(num_classes=5, sparse_labels=True, weightage="quadratic")
     check_metric_serialization(ck, actuals, preds, weights)
+
+def test_cohen_kappa_single_batch():
+    # Test for issue #1962
+    obj = CohenKappa(5, regression=True, sparse_labels=True)
+
+    try:
+        # Test single batch update
+        obj.update_state(tf.ones(1), tf.ones(1))
+    except:
+        pytest.fail("Unexpected exception")
```

Comment:
Why we have to use `try` and `except` here? It should work after the fix. So it might be something like
```python
def test_cohen_kappa_single_batch():
    obj = CohenKappa(5, regression=True, sparse_labels=True)
    obj.update_state(tf.ones(1), tf.ones(1))
    np.testing.assert_allclose(xxx, obj.result())
```
where `xxx` is the numeric result.

Reply:
Ok. I'll take this approach. I added the assertion value check in my last commit, but for a "perfect" Kappa, it return `nan` (I double-checked if this was an issue, but even sklearn implementation has the same result, due to denominator being equal to zero).
The solution was to call `obj.update_state(tf.ones(1), tf.zeros(1))` instead, which results in zero kappa value.